### DISCUSSION TITLE
How to cascade delete record using python package or API ?

### DISCUSSION THREAD

**Discussion by Spritan** [supabase/supabase#28117] [Questions]
How to cascade delete record using python package or API ?

**Comment by GaryAustin1** [comment]
Cascade delete is set up in SQL on the column in the database.  You can use the UI to set it up, or SQL editor.
https://supabase.com/docs/guides/database/postgres/cascade-deletes

Edit:  It is an automatic thing once set up.  You don't control it with the delete operation itself.
